packages/@react-spectrum/test-utils/package.json

```diff
@@ -28,7 +28,7 @@
     "@swc/helpers": "^0.5.0"
   },
   "peerDependencies": {
-    "@testing-library/react": "^13.0.0 || ^14.0.0",
+    "@testing-library/react": "^15.0.0",
```

Comment:
given the difference in aria-query, i don't think we can support all these versions